Don't create /var/lib/soju, it's done by soju.pre-install script
with proper permissions.

diff --git a/Makefile b/Makefile
index bcf4e07..3f210f7 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,6 @@ install: all
 	mkdir -p $(DESTDIR)$(PREFIX)/$(BINDIR)
 	mkdir -p $(DESTDIR)$(PREFIX)/$(MANDIR)/man1
 	mkdir -p $(DESTDIR)/etc/soju
-	mkdir -p $(DESTDIR)/var/lib/soju
 	cp -f soju sojuctl $(DESTDIR)$(PREFIX)/$(BINDIR)
 	cp -f doc/soju.1 $(DESTDIR)$(PREFIX)/$(MANDIR)/man1
 	cp -f config.in $(DESTDIR)/etc/soju/config
